docs/auth_credentials.md

```diff
@@ -1,6 +1,14 @@
 # Authentication credentials options
 
-Note: Only one type of credential can be used at a time. If multiple credentials are provided, the last one added will take precedence.
+> [!NOTE]
+> Only one type of credential can be used at a time. If multiple credentials are provided, the last one added will take precedence.
+
+**Choosing the right credential type:**
```

Comment:
I disagree with most of the guidance below here can we skip it this round @manny-silva ? I agree guidance would be good but this is not quite right.

Reply:
I think it's more nuanced and would want Ashley and Daniel to weigh in, tbh.